src/Components/creatorPage/AuthorInfo.js

```diff
@@ -30,8 +30,14 @@ export default function AuthorInfo({
 }
 
```

Comment:
Can you rename this file (and the author.css file) to use creator instead of author?